centaur/src/main/resources/standardTestCases/wdl_biscayne/biscayne_http_relative_imports/biscayne_http_relative_imports.wdl

```diff
@@ -0,0 +1,12 @@
+version biscayne
+
+# Is there a better way to test this?
+import "https://raw.githubusercontent.com/broadinstitute/cromwell/cjl_relative_imports/womtool/src/test/resources/validate/biscayne/valid/relative_imports/sub_wfs/foo.wdl"
```

Comment:
You could look into using a mock server to serve a specific URL off of localhost

Example of how we test Rawls pulling from Dockstore without actually calling Dockstore:
https://github.com/broadinstitute/rawls/blob/develop/core/src/test/scala/org/broadinstitute/dsde/rawls/mock/RemoteServicesMockServer.scala#L259

Reply:
Or right in Cromwell for that matter: https://github.com/broadinstitute/cromwell/blob/develop/languageFactories/wdl-draft2/src/test/scala/languages.wdl.draft2/WdlWorkflowHttpImportSpec.scala#L41

Reply:
I think this will definitely work in the unit tests.

I wonder if centaur can be changed to host its own set of http files in some non case-by-case manner? As we move more towards HTTP based files, I think that'll become more and more useful